app/javascript/article-form/components/Help/Format.jsx

```diff
@@ -0,0 +1,135 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+const Format = ({ show, toggleModal }) => {
```

Comment:
refactor (non-blocking): Consider a better name for the component. I'm not sure what the `<Format />` component is by looking at it. Looking at the JSX within, I can deduce it's for displaying help. Maybe `<EditorHelp />` or `<EditorFormattingHelp />`